ark/segmentation/marker_quantification.py

```diff
@@ -266,14 +266,17 @@ def generate_cell_table(segmentation_labels, tiff_dir, img_sub_folder,
 
     # if no fovs are specified, then load all the fovs
     if fovs is None:
-        # handle mibitiffs with an assumed file structure
         if is_mibitiff:
-            filenames = io_utils.list_files(tiff_dir, substrs=['.tif'])
-            fovs = io_utils.extract_delimited_names(filenames, delimiter=None)
-        # otherwise assume the tree-like directory as defined for tree loading
+            fovs = io_utils.list_files(tiff_dir, substrs=['.tif'])
         else:
-            filenames = io_utils.list_folders(tiff_dir)
-            fovs = filenames
+            fovs = io_utils.list_folders(tiff_dir)
+
+    # drop file extensions
+    fovs = io_utils.remove_file_extensions(fovs)
+
+    # if mibitiff, also need to remove delimiters
+    if is_mibitiff:
+        fovs = io_utils.extract_delimited_names(fovs)
```

Comment:
I think this is going to lead to unexpected results. There's no particular reason that people using MIBITIFF files will definitely want any underscores removed from their FOV names, especially if it's buried inside this function and can't be modified. 

In fact, I think if you set MIBITIFF=True in the notebook, this will fail, since we don't do this delimiter removal when making the segmentation_labels.xr, which is what the FOVs are being compared to.

Reply:
@ngreenwald confirmed that setting `MIBITiff = True` will fail because the actual `fov` coordinate being set is that of the entire `fov8_Row8Depth8...`.

I can remove these two lines, but as a follow-up question, for MIBITiff inputs, can we safely assume that the coordinates  will always be that of the entire fov name (as in, `fov8_Row8Depth8...`)?

Reply:
No, I think people will have all kinds of creative ways of naming their mibitiffs. Hence, defaulting the delimiter to None will just pass it through unmodified. If people want to clean up their FOV names, they can either specify the correct, custom delimiter at each step in the pipeline, or clean them up prior to starting.